src/main/java/org/bitbucket/nanojava/NanoJava.java

```diff
@@ -0,0 +1,256 @@
+/* Copyright (C) 2011-2020  Egon Willighagen <egonw@users.sf.net>, Ammar Ammar <ammar257ammar@gmail.com>
```

Comment:
This is a new file and I did not contribute, so should not be listed as copyright owner.

Reply:
Oh, also, this file is started in 2020, so list only 2020.

Reply:
Done